### PR TITLE
fix: new lints coming with 1.78 and 1.80-nightly

### PR DESCRIPTION
**Pull request by zvolin**

### CODE REVIEW

File: types/Cargo.toml [hunk 1]
```diff
@@ -57,8 +57,7 @@ test-utils = ["dep:ed25519-consensus", "dep:rand"]
 wasm-bindgen = ["celestia-tendermint/wasm-bindgen"]
 
 [package.metadata.docs.rs]
-features = ["p2p", "test-utils"]
```
[the remark this answers is not in the document]
Comment (zvolin) [reply types/Cargo.toml:60]: thanks, it's by accident, I was preparing an example for cargo issue :sweat_smile:

[the remark this answers is not in the document]
Comment (zvolin) [reply types/Cargo.toml:60]: fixed

File: node-wasm/src/utils.rs [hunk 2]
```diff
@@ -91,15 +86,4 @@ where
...
-    fn with_js_context<F, C>(self, context_fn: F) -> Result<T, JsError>
-    where
-        C: fmt::Display + Send + Sync + 'static,
-        F: FnOnce() -> C,
-    {
-        self.map_err(|e| {
-            let context = context_fn();
-            JsError::new(&format!("{context}: {e}"))
-        })
-    }
```
Comment (oblique) [inline node-wasm/src/utils.rs:104]: What was the issue here?

Comment (zvolin) [reply node-wasm/src/utils.rs:104]: Unused

Comment (zvolin) [reply node-wasm/src/utils.rs:104]: I can keep it, but thought that we can easily add it when we need it

**Review by fl0rek — APPROVED** [review]
nice!

**Pull request merged**